.github/workflows/docker-build.yml

```diff
@@ -346,14 +355,14 @@ jobs:
           npm install snyk -g
           snyk -v
           snyk auth ${{ env.SNYK_TOKEN }}
-          snyk container test ${{ inputs.image_registry }}/${{ inputs.image_organization }}/${{ inputs.image_name }}:${{ inputs.tag }} --file=${{ inputs.dockerfile }} --severity-threshold=critical --target-reference=${{ inputs.snyk_target_ref }}
```

Comment:
This allows any dominant tag to be scanned by snyk instead of only the one entered through the single tag input.